packages/notebook/src/widget.ts

```diff
@@ -1207,13 +1207,28 @@ export class Notebook extends StaticNotebook {
   /**
    * Construct a notebook widget.
    */
+
   constructor(options: Notebook.IOptions) {
     super(Private.processNotebookOptions(options));
     this.node.tabIndex = 0; // Allow the widget to take focus.
     // Allow the node to scroll while dragging items.
     this.node.setAttribute('data-lm-dragscroll', 'true');
+    this.activeCellChanged.connect(this._updateSelectedCells, this);
+    this.selectionChanged.connect(this._updateSelectedCells, this);
+  }
+
+  get selectedCells(): Cell[] {
+    return this._selectedCells;
   }
 
+  private _updateSelectedCells(): void {
+    this._selectedCells = this.widgets.filter(cell =>
+      this.isSelectedOrActive(cell)
+    );
+  }
+
+  private _selectedCells: Cell[] = [];
```

Comment:
Could you move this line at the end of the class with all the other private attribute definitions?

Reply:
This has been done.